app/Http/Controllers/Api/PublisherController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Models\User;
+use App\Models\Publisher;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+use App\Http\Controllers\Controller;
+
+class PublisherController extends Controller
+{
+    public function createPublisher(Request $request)
```

Comment:
@radosevic-dr Functions should have comments

Reply:
Function is about creating Publisher, actually I don't know what to comment